templates/trust-private-ca-on-all-nodes/containerd-ca-update-script.yaml

```diff
@@ -0,0 +1,95 @@
+################################
+## Containerd toml ConfigMap  ##
+################################
+{{- if and .Values.global.privateCaCertsAddToHost.enabled .Values.global.privateCaCertsAddToHost.addToContainerd }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-cert-copy-and-toml-update
+  labels:
+    component: houston
+    tier: astronomer
+    release: {{ .Release.Name }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: {{ .Release.Service }}
+data:
+  update-containerd-certs.sh: |
+    #!/usr/bin/env sh
+    if [ ! -f /hostcontainerd/config.toml ]; then
```

Comment:
theoretically I think this file could be totally absent and need to be created, but if we have tested on Azure, AWS, and GCP I'm fine with crossing that bridge when we come to it. In reality, I would expect it always exists.

Reply:
yes this file always exists in all cloud provider only difference is the content